frequency-chain/provider-dashboard
# New logo & favicon, size and placement adjustments

## Goal
* Reduce margin-bottom between logo & header from 14 to 8
* Use new favicon
* Use new logo, reduce size/width to match "Provider Dashboard"

Closes #171, and "Replace favicon" draft issue in the project.  The issue couldn't be converted because I guess it doesn't like having to "transfer assets"

## Checklist


## How To Test the Changes

1. Clone the pr branch
2. ```shell
    npm install && npm run dev
   ```
3. Verify that the new logo and favicon are present. Perhaps turn on responsive viewing in Dev console & check that there are no issues at mobile sizes.

<img width="909" alt="Screenshot 2024-09-26 at 3 22 40 PM" src="https://github.com/user-attachments/assets/55654d42-fd95-4c86-b4a1-610472f1d227">
